source/common/less/theme-berlin/dark.less

```diff
@@ -123,7 +123,7 @@ body.dark {
     .cm-link,
     .cm-strong,
     .cm-em {
-      color: var(--c-primary-shade);
+      color: var(--grey-0);
```

Comment:
If you want `var(--grey-0)` as the colors, you can completely remove these CSS statements, since they will inherit the font color from the top level (see line 11).

The same happens in light mode.

Reply:
:+1: